README.md

```diff
@@ -19,3 +20,10 @@ quantitatively reason with them.
 Add your answer to this markdown file. [This
 page](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions)
 might help with the notation for mathematical expressions.
+
+
+## My Response
+
+When it comes to choosing the best pivot for a quicksort algorithm there are two qualities you look for in a pivot. The first quality is a pivot which is easy to choose, and the second is one that pivots to the middle of the array. So when choosing to use a random pivot or choosing the median of three we have to consider a few factors to find which is better. The way I see it in order to figure out which is better we need to find out which one has a higher chance of being a good pivot.
+
+The way I would describe a good pivot would be the middle third of the array. So a good pivot would be greater than 1/3 but less than 2/3. So all we need to do is figure out which one has a better chance of being in the middle of the array. The random one is easy since it has a 1 in three chance of being a good choice, but the middle of three is a bit harder to figure out. So to figure out the odds of the choice to being good I just came up with all possible combinations and then counted all the times that a good pivot would be in the middle. This gave me a 3/8 chance to pick a good pivot. Therefore I believe picking the median of three numbers leads to a better outcome more of the time.
```

Comment:
3/8 is less than 1/2 -- so median of three is worse than just choosing one element?